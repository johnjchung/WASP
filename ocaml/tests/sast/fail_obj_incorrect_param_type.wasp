Object Person() {
	int age;
	int id; 
}

Endpoint badCitizen(GET:) {
    Person p1 = Person { 10, 15.5 };
}

