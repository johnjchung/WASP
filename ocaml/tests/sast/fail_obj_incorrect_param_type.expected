Fatal error: exception Semantic_check.ObjectArgTypeMismatchError("ARG TYPE MISMATCH ERROR ON OBJECT LITERAL TO OBJECT Person")
